superset/utils/csv.py

```diff
@@ -65,3 +67,20 @@ def df_to_escaped_csv(df: pd.DataFrame, **kwargs: Any) -> Any:
     df = df.applymap(escape_values)
 
     return df.to_csv(**kwargs)
+
+
+def get_chart_csv_data(
+    chart_url: Optional[str], auth_cookies: Optional[Dict[str, str]] = None
```

Comment:
Why is this `Optional` ?

Reply:
good catch. This doesn't need to be optional. I will remove that.